es-app/src/SystemScreenSaver.cpp

```diff
@@ -145,24 +174,16 @@ void SystemScreenSaver::startScreenSaver()
 			pickRandomGameListImage(path);
 		}
 
-		if (!mImageScreensaver)
+		// Check if file has a known video extension
+		std::string pathExtension = path.substr(path.find_last_of(".") + 1);
+		std::vector<std::string> videoExtensions {"mp4", "avi"};
```

Comment:
This should probably be defined outside of the function.

Reply:
That makes a lot more sense, resolved in db2ce15